20181004-demo-sprint-4.md

```diff
@@ -8,14 +8,60 @@ Slides: [ZDS2 Demo Spint 4.pdf](/community/bestanden/zds2-demo-sprint-4.pdf)
+totdo
+
+#### Hamvraag is er een andere manier om dit op te lossen en is die beter?
+todo
+
+#### Feedback
+Ivo van Zitteren (Breda) doet de suggestie om middels autorisatie/authenticatie te borgen dat gegevens schoon blijven.
+Jeffrey Gortmaker (VNG) geeft aan dat er grofweg 3 oplossingen zijn en verzoekt het publiek hier over na te denken en hier zo snel mogelijk op te reageren.
+
+#### Als ik wil reageren op vraag van Jeffrey, hoe/waaar doe ik dat? Welk issue is dat?
+todo
+
+### Kan het patroon ook zonder synchronisatie gebruikt kan worden? 
+Dit wordt bevestigd en het is zaak een sweetspot te vinden tussen alles afdekken en een risico dat dingen uit pas gaan lopen.
+
+#### De logica om te valideren of zaaktype klopt ligt nu bij de ZRC, waarom ligt die niet bij ZTC?
+Het is de verantwoordelijkheid van ZRC om het gebruikte zaaktype te valideren. Andy Verberne (ATOS) is het er niet helemaal mee eens. 
+
+#### Feedback/opmerkingen
+(Eelco Hotting) In de toekomst gaan we waarschijnlijk gebruik maken van verschillende ZTC's, bijvoorbeeld per domein of proces. Theo Peters (VNG) vult aan: Bijvoorbeeld in het ruimtelijk domein is het interessant om één gezamelijke, gedeelde ZRC en ZTC te gebruiken. In het Sociaal domein willen we dat misschien liever lokaal houden.
+(principe CG om technisch mogelijk te maken dat dit kan
+ 
+#### Deze manier van werken vereist business logca, niet meer alleen opslaan zoals nu in de standaard staat.
+Dit klopt inderdaad.Door deze business logica te beschrijven wordt het gemakkelijker bovenop deze standaarden te ontwikkelen. Punt van aandacht is wel dat er gradaties zijn van wat een gemeente verstaat onder "zaakgericht werken". Omdat er nooit goed opgeschreven is wat er wel/niet onder verstaan wordt is een doelstelling om deze onduidelijkheid weg te nemen. In ieder geval moet de kern vastgelegd worden waar we het over eens zijn waarbij het belangrijk is de consistentie regels op te nemen.
+
+#### Hoe worden designkeuzes bepaald en vastgelegd?
+Dit zijn we nog aan het ontdekken. Weliswaar zit het in de mens om hiervoor een procedure voor te bedenken ("zo gaan we het doen") maar het belangrijkste is elkaar op te zoeken en te overleggen waar nodig is.
+
+#### Wordt een Besluit niet meer gezien als een subtype van een Informatieobject?
+Een Besluit is een zelfstandig concept, een Informatieobject is een manier om het Besluit kenbaar te maken maar een ander concept.
+
+#### Zijn bijvoorbeeld informatiemanagers geïnteresseerd in het soort documentatie zoals die nu gemaakt wordt?
+Antwoord: Dat is precies de vraag die wij proberen te (laten) beantwoorden. In de werkgroep zitten veelal techneuten en inhoudsdeskundigen waardoor het lastig in te schatten is of documentatie diep genoeg of juist niet te diep gaat en of niet teveel kennis verondersteld wordt aanwezig te zijn. De documentatie die er nu is wordt gegenereerd uit de OAS en is voor niet-technische mensen begrijpelijker dan de OAS.
```

Comment:
M.i. toevoegen: "We werken aan functionele documentatie, onder meer om zicht te krijgen op de behoefte daaraan en invulling daarvan."